radio/src/main_avr.cpp

```diff
@@ -129,6 +132,9 @@ void perMain()
   const char *warn = s_warning;
   uint8_t menu = s_menu_count;
 
+#if defined(LCD_ST7920)
+  if(0==lcdstate){//No need to redraw until lcdRefresh_ST7920(0) below completely refreshes the display.
```

Comment:
Somewhere in board_9x.h you define a new macro like this:
```
#if defined(LCD_ST7920)
#define IS_LCD_REFRESH_ALLOWED() (0==lcdstate)
#else 
#define IS_LCD_REFRESH_ALLOWED() (1)
#endif 
```
So that here you can remove all these new #ifdef with:
```
if (IS_LCD_REFRESH_ALLOWED()) {
...
}
```